.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+
+jobs:
+  rspec:
+    parameters:
+      ruby-version:
+        type: string
+      gemfile:
+        type: string
+    docker:
+      - image: circleci/ruby:<< parameters.ruby-version >>-node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install --gemfile=gemfiles/<< parameters.gemfile >>.gemfile
+      - run:
+          name: Setup test app
+          command: bundle exec rake test_app
+      - run:
+          name: Run RSpec
+          command: bundle exec rake spec
+
+workflows:
+  all-specs:
+    jobs:
+      - rspec:
+          matrix:
+            parameters:
+              gemfile:
+                - spree_3_1
+                - spree_3_7
+                - spree_4_0
+                - spree_4_1
+                - spree_4_2
+                - spree_master
```

Comment:
```
                - spree_4_3
                - spree_main
```

Reply:
I don't know if I should create `spree_4_3.gemfile` by myself or generate it somehow, but is this file content ok? :point_down: 

```
source "https://rubygems.org"

gem "rails-controller-testing"
gem "rubocop", "~> 0.63.0", require: false
gem "rubocop-rspec", require: false
gem "spree_auth_devise", "~> 4.3.0"
gem "spree_backend", "~> 4.3.0"
gem "spree_core", "~> 4.3.0"
gem "sass-rails"
gem "rspec-rails", "~> 4.0.0.beta2"

gemspec path: "../"

```